db/models/Video.go

```diff
@@ -22,6 +22,7 @@ type Video struct {
 	User        *User  `json:"user"`
 	Title       string `json:"title"`
 	Description string `json:"description"`
+	Thumbnail   uint64 `json:"thumbnail"`
```

Comment:
`Thumnail` is also file, so we need to provide file id there. The same way, as you did in User.go